### ISSUE TITLE
Filter API inconsistencies 

### ISSUE THREAD

**Issue by smthomas** [wp-graphql/wp-graphql#1588]
In the `graphql_map_input_fields_to_wp_query` filter, the $post_type is the last parameter. This is inconsistent with the other `graphql_map_input_fields_*` filters. For example in `graphql_map_input_fields_to_get_terms` the $taxonomy parameter is the third parameter (while the $context & $info parameters are the last two in all of the `graphql_map_input_fields_*` filters). For consistency sake, it may make sense to update this filter.

wp_query filter: https://github.com/wp-graphql/wp-graphql/blob/560bd4ab94ffbc939014a90c4ffa9ca5da1880fa/src/Data/Connection/PostObjectConnectionResolver.php#L431

get_terms filter: https://github.com/wp-graphql/wp-graphql/blob/560bd4ab94ffbc939014a90c4ffa9ca5da1880fa/src/Data/Connection/TermObjectConnectionResolver.php#L225

**Comment by justlevine** [comment]
Related: #331

**Comment by justlevine** [comment]
Removing `not stale`, now that stalebot is behaving.

While filter consistency is still an issue, the specific inconsistencies will need to be reviewed before we can figure out an implementation plan (and what can be addressed via breaking/nonbreaking changes).

**Comment by tihom4537** [comment]
I have generated a pull request. Do consider it for the review .

**Comment by justlevine** [comment]
Hi @tihom4537 

The effort is appreciated. However, as I mentioned [above](https://github.com/wp-graphql/wp-graphql/issues/1588#issuecomment-1837441050), this is currently on hold. It likely won't be merged until we start working on v2.0, and that's assuming no other changes invalidate it.
